code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -194,6 +209,9 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 		cached_gases[id][MOLES] -= removed_gases[id][MOLES]
 	garbage_collect()
 
+	removed.heat_capacity = ratio * heat_capacity
+	heat_capacity *= (1 - ratio)
```

Comment:
Possible tracked heat capacity drift due to the quantize above. Why do we quantize here?